scripts/gen_vimdoc.py

```diff
@@ -1114,7 +1118,7 @@ def main(config, args):
             docs += '\n\n\n'
 
         docs = docs.rstrip() + '\n\n'
-        docs += ' vim:tw=78:ts=8:ft=help:norl:\n'
+        docs += ' vim:tw=78:ts=8:sw=4:sts=4:et:ft=help:norl:\n'
```

Comment:
if `indentation` is a variable now, might as well use that instead of hardcoding?